ftp/1.5/modules/ROOT/pages/ftp-troubleshooting.adoc

```diff
@@ -0,0 +1,39 @@
+= Troubleshoot FTP Connector - Mule 4
+
+To troubleshoot Anypoint Connector for FTP (FTP Connector), become familiar with the information about troubleshooting Mule application issues.
+
+== Troubleshoot Mule Application Issues
+
+Sometimes you are not sure why the application is failing and you require a better understanding of what is going on in the app's interaction with the FTP server.
+
+To resolve this error:
+
+* Use the configuration file to increase the logging level on the server side.
+
+* Try to replicate the issue using an FTP client, for example, Cyberduck or Filezilla. If the issue also occurs with these clients, then it is likely that the issue is not related to the Mule app.
+
+* Get better and more detailed logs on the Mule app side by enabling verbose logging for FTP Connector and for the JSCH driver:
++
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. If the following line is already in the `log4j2.xml` file, uncomment it to enable it; otherwise, add it:
++
+[source,xml,linenums]
+----
+    <AsyncLogger name=“org.mule.extension.ftp” level=“TRACE”/>
+    <AsyncLogger name=“org.apache.commons.net” level=“DEBUG”/>
+----
+[start=6]
+. Add the following argument to your run configuration:
++
+ `-M-Djavax.net.debug=ssl,handshake`
+
+* Try to minimize or remove unnecessary DataWeave data transformations (you could even use hard coded data) to minimize app's complexity and determine whether the issue is with the interaction between the app and the FTP server.
+
+To get further information about enabling verbose logging for SDK components in your Mule application, see the xref:mule-sdk::troubleshooting.adoc[troubleshoot SDK components] documentation.
```

Comment:
Consider changing "To get further information about" to "For additional information about"